### PR TITLE
mrc-2718 Fix massive runWorkflowTabs bundle

### PR DESCRIPTION
**Pull request by hillalex**
Although the Bootstap-Vue [docs](https://bootstrap-vue.org/docs#individual-components-and-directives) imply that the current syntax should only import the desired component, the behaviour seems to be that it's pulling in the whole library. This change imports just the alert component where needed.

### CODE REVIEW

**Review by EmmaLRussell — COMMENTED** [review]
This runs fine manually but looks like tests are failing.

**Comment by hillalex** [comment]
Sorry, I must have been in some kind of norovirus induced delirium when I added you to this clearly not ready PR!

File: src/app/static/jest.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
     "globals": {
         "appUrl": "http://app",
         "ts-jest": {
-            tsConfig: 'tsconfig.json',
+            "tsconfig": 'tsconfig.json',
```
Comment (hillalex) [inline src/app/static/jest.config.js:10]: opportunistic fix - jest was warning about `tsConfig` being deprecated in favour of `tsconfig`

File: src/app/static/babel.config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ['@babel/preset-env'],
```
Comment (hillalex) [inline src/app/static/babel.config.js:2]: why does changing `.babelrc` to `babel.config.js` work? no-one knows

**Review by EmmaLRussell — APPROVED** [review]
Workflow bundle is down to 87KB and alert working as expected. :+1:

**Review by carbowizzy — APPROVED** [review]
LGTM

**Pull request merged**